docs/sources/reference/components/otelcol.exporter.debug.md

```diff
@@ -0,0 +1,164 @@
+If necessary, bug reports or feature requests are redirected to the upstream repository.
+{{< /admonition >}}
+
+Multiple `otelcol.exporter.debug` components can be specified by giving them different labels.
+
+## Usage
+
+```river
+otelcol.exporter.debug "LABEL" { }
+```
+
+## Arguments
+
+`otelcol.exporter.debug` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`verbosity`           | `string` | Verbosity of the generated logs. | `"normal"` | no
+`sampling_initial`    | `int`    | Number of messages initially logged each second. | `2` | no
+`sampling_thereafter` | `int`    | Sampling rate after the initial messages are logged. | `500` | no
+
+The `verbosity` argument must be one of:
+* `"basic"`: A single-line summary of received data is logged to stderr, with a total count of telemetry records for every batch of received logs, metrics or traces.
+* `"normal"`: Currently, this produces the same output as `"basic"` verbosity.
+* `"detailed"`: All details of every telemetry record are logged to stderr, typically writing multiple lines for every telemetry record.
+
+Example of `"basic"` and `"normal"` output:
+```
+ts=2024-06-13T11:24:13.782957Z level=info msg=TracesExporter component_path=/ component_id=otelcol.exporter.debug.default "resource spans": 1, spans: 2
+```
+
+Example of `"detailed"` output:
+```
+ts=2024-06-13T11:24:13.782957Z level=info msg=TracesExporter component_path=/ component_id=otelcol.exporter.debug.default "resource spans"=1 spans=2
+ts=2024-06-13T11:24:13.783101Z level=info msg="ResourceSpans #0
+Resource SchemaURL: https://opentelemetry.io/schemas/1.4.0
+Resource attributes:
+     -> service.name: Str(telemetrygen)
+ScopeSpans #0
+ScopeSpans SchemaURL:
+InstrumentationScope telemetrygen
+Span #0
+    Trace ID       : 3bde5d3ee82303571bba6e1136781fe4
+    Parent ID      : 5e9dcf9bac4acc1f
+    ID             : 2cf3ef2899aba35c
+    Name           : okey-dokey
+    Kind           : Server
+    Start time     : 2023-11-11 04:49:03.509369393 +0000 UTC
+    End time       : 2023-11-11 04:49:03.50949377 +0000 UTC
+    Status code    : Unset
+    Status message :
+Attributes:
+     -> net.peer.ip: Str(1.2.3.4)
+     -> peer.service: Str(telemetrygen-client)
+Span #1
+    Trace ID       : 3bde5d3ee82303571bba6e1136781fe4
+    Parent ID      :
+    ID             : 5e9dcf9bac4acc1f
+    Name           : lets-go
+    Kind           : Client
+    Start time     : 2023-11-11 04:49:03.50935117 +0000 UTC
+    End time       : 2023-11-11 04:49:03.50949377 +0000 UTC
+    Status code    : Unset
+    Status message :
+Attributes:
+     -> net.peer.ip: Str(1.2.3.4)
+     -> peer.service: Str(telemetrygen-server)
+        {"kind": "exporter", "data_type": "traces", "name": "debug"}"
+```
+
+Note that the above `"detailed"` example was edited to replace all instances of `\n` with a new line.
```

Comment:
Is this important in the context of the example? If a user looks at this output in a terminal will they see the `\n`? Will the detailed output always have `\n`?

Reply:
Yes, they will always see `\n` instead of actual new lines. They need to search and replace the `/n` manually.

Reply:
```suggestion
{{< admonition type="note" >}}
All instances of `\n`  in the `"detailed"` example have been replaced with new lines.
{{< /admonition >}}
```